docs/deployer-tool.md

```diff
@@ -0,0 +1,24 @@
+# Deploy/Upgrade RP
+
+`deploy` directory contains artifacts for manual environment deployment.
+Production deployment and upgrade can be orchestrated using deployment tooling:
+`go run ./cmd/aro deployer`
+
+Utility is decoupled from `env` package and requires these variables to run:
+```
+AZURE_TENANT_ID - Required
+AZURE_CLIENT_SECRET - Required
+AZURE_CLIENT_ID - Required
+AZURE_SUBSCRIPTION_ID - Required
+AZURE_RP_PARAMETERS_FILE - Required, location of environment parameters file
+AZURE_RP_RESOURCEGROUP_NAME - Required, RP resource group name
+AZURE_RP_VERSION - overrides gitCommit version for the RP VMSS
```

Comment:
in a follow-up PR, start to get these all together into a single manifest file which can be read on startup

Reply:
Why not use the parameter file as a format? know, arm validates it and less transformation and change?